### DISCUSSION TITLE
Why OpenVino EP faster than CPU EP?

### DISCUSSION THREAD

**Discussion by PhilippShemetov** [microsoft/onnxruntime#13389] [Other Q&A]
Hello, I have a question. Why is OpenVino EP (CPU F32) faster than ONNXRuntime CPU EP without GRAPH_OPTIMIZATION? What is going on inside OpenVino? Because I only see tensor transformations and fusions in the OpenVino code and I can't find the part of the code that is responsible for optimization.

**Comment by faxu** [comment]
You can read more about the OpenVINO EP here: https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html

If using ORT on Intel devices with OpenVINO, it's expected that the OpenVINO EP would be optimized for the hardware and perform better than the default CPU provider.
